### PR TITLE
feat: add video-listing page

### PR DESCRIPTION
**Pull request by anik31**
## Video Listing Page
I have added the Video Listing page. Ignore the index files. Focus only on the components, pages & context + reducer files.
 
## Files to check
- [ ]  src/context/video-context.js
- [ ]  src/pages/home/CategoryCard.jsx
- [ ]  src/pages/video-listing/CategoryChips.jsx
- [ ]  src/pages/video-listing/Videos.jsx
- [ ]  src/pages/video-listing/videos.css
- [ ]  src/reducer/video-reducer.js
- [ ]  src/utils/filterVideos.js
 
 _PS - Ignore other files._
 
https://user-images.githubusercontent.com/56336326/160052476-c3a0fb3c-8760-445f-b6aa-3023debbc241.mp4

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -18,8 +18,10 @@ export function Navbar(){
         </nav>
         <nav className="navigation">
             <ul>
-                <li><Link to="/explore" className="btn btn-secondary-link">Explore</Link></li>
+                <li><Link to="/videos" className="btn btn-secondary-link">Explore</Link></li>
```
Comment (hsnice16) [inline src/components/Navbar/Navbar.jsx]: As these list items have same styling, so you can map over an array and can return these list.

Like,
```javascript
[{linkTo: "/videos", linkFor: "Explore"}, ...other objects].map(({linkTo, linkFor}) => <li 
to={linkTo} className="...">{linkFor}</li>)
```

**Review by hsnice16 — APPROVED** [review]
UI is superb. Great work on that. The code is nice too. Have added a few suggestions, nothing more to suggest.

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (anik31) [reply src/components/Navbar/Navbar.jsx]: Thanks for the suggestion @hsnice16 , I will update it.

**Review by abhisheksahani-nova — APPROVED** [review]
Great work with the clean and readable code , looks good to merge

**Pull request merged**